First Release PR - Pacvar (#4)
nf-core/pacvar: docs/usage.md

```bash
--input '[path to samplesheet file]'
```

| Column   | Description                                             |
| -------- | ------------------------------------------------------- |
| `sample` | Name of the sample.                                     |
| `bam`    | Path to the unaligned bam file`.                        |
| `pbi`    | Path to the associated pbi - bam index file. (optional) |

### Example samplesheet

Each sample corresponds to a different run on PacBio revio machines. Each sample will (if not skipping demultiplexing) will be demultiplexed according to the input `barcodes.fasta`, therefore each sample in the samplesheet should have the same adapters.

```csv title="samplesheet.csv"
sample,bam,pbi
CONTROL_REP1,NA03697B2_01.bam,NA03697B2_01.bam, NA03697B2_01.bam,NA03697B2_01.pbi
CONTROL_REP1,NA03697B2_02.bam,NA03697B2_02.bam, NA03697B2_02.bam,NA03697B2_02.pbi
```

## Running the pipeline

The typical command for running the pipeline is as follows for the wgs and the repeat workflow:

```bash

> nextflow run nf-core/pacvar --input ./samplesheet.csv --outdir ./results --genome 'GATK.GRCh38' -profile docker --barcodes barcodes.fasta --intervals intervals.bed --workflow wgs --snv_caller deepvariant

Reviewer: ```suggestion
nextflow run nf-core/pacvar --input ./samplesheet.csv --outdir ./results --genome 'GATK.GRCh38' -profile <docker/singularity/conda> --barcodes barcodes.fasta --intervals intervals.bed --workflow wgs --snv_caller deepvariant
```

> nextflow run nf-core/pacvar --input ./samplesheet.csv --outdir ./results --genome  'GATK.GRCh38' -profile docker --barcodes barcodes.fasta --intervals intervals.bed --workflow repeat --repeat_id repeat-id --snv_caller 'deepvariant'

Reviewer: ```suggestion
nextflow run nf-core/pacvar --input ./samplesheet.csv --outdir ./results --genome  'GATK.GRCh38' -profile <docker/singularity/conda> --barcodes barcodes.fasta --intervals intervals.bed --workflow repeat --repeat_id repeat-id --snv_caller 'deepvariant'
```

```

> This will launch the pipeline with the `docker` configuration profile. See below for more information about profiles.

Reviewer: ```suggestion
> See below for more information about profiles.
```

## Introduction

The nf-core/pacvar pipeline has been designed to analyze data from multiple runs, gathering its input from a provided samplesheet.

## Samplesheet input

You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this `--input` parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.

```bash
--input '[path to samplesheet file]'
```

| Column   | Description                                             |
| -------- | ------------------------------------------------------- |
| `sample` | Name of the sample.                                     |

> | `bam`    | Path to the unaligned bam file`.                        |

Author: fixed

--input '[path to samplesheet file]'
```

| Column   | Description                                  |
| -------- | -------------------------------------------- |
| `sample` | Name of the sample.                          |
| `bam`    | Path to the unaligned bam file`.             |

> | `bai`    | Path to the associated bai - bam index file. |

Reviewer 2: pbi?

> ### Example SampleSheet

Reviewer 2: ```suggestion
### Example samplesheet
```

Each sample cooresponds to a different run on PacBio revio machines. Each sample will (if not skipping demultiplexing) will proceed to be demultiplexed according to the inputted --barcodes barcodes.fasta, therefore each sample in the samplesheet should have the same adapaters.

```csv title="samplesheet.csv"
sample,bam,pbi

> CONTROL_REP1, NA03697B2_01.bam,NA03697B2_01.pbi
> CONTROL_REP1, NA03697B2_02.bam,NA03697B2_03.pbi

Reviewer 2: ```suggestion
CONTROL_REP1,NA03697B2_01.bam,NA03697B2_01.pbi
CONTROL_REP1,NA03697B2_02.bam,NA03697B2_02.pbi
```

```

An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.

## Running the pipeline

The typical command for running the pipeline is as follows for the wgs and the repeat workflow:

```bash
nextflow run nf-core/pacvar --input ./samplesheet.csv --outdir ./results --genome GRCh37 -profile docker --barcodes barcodes.fasta --intervals intervals.bed --workflow wgs

> nextflow run nf-core/pacvar --input ./samplesheet.csv --outdir ./results --genome GRCh37 -profile docker --barcodes barcodes.fasta --intervals intervals.bed --workflow repeat --id repeat-id --tools 'deepvariant'

Reviewer 2: Does this mean only one repeat is analysed at the time?

Author: yes - for now, but I agree in later versions I'll work on making it analyze more than one repeat region

### Example samplesheet

> Each sample cooresponds to a different run on PacBio revio machines. Each sample will (if not skipping demultiplexing) proceed to be demultiplexed according to the inputted --barcodes barcodes.fasta, therefore each sample in the samplesheet should have the same adapaters.

Reviewer: ```suggestion
Each sample corresponds to a different run on PacBio revio machines. Each sample will (if not skipping demultiplexing) will be demultiplexed according to the input `barcodes.fasta`, therefore each sample in the samplesheet should have the same adapters.
```
